apps/editing-toolkit/webpack.config.js

```diff
@@ -74,6 +75,14 @@ function getWebpackConfig( env = {}, argv = {} ) {
 			...webpackConfig.plugins.filter(
 				( plugin ) => plugin.constructor.name !== 'DependencyExtractionWebpackPlugin'
 			),
+			new webpack.DefinePlugin( {
+				'process.env.NODE_ENV': JSON.stringify( process.env.NODE_ENV ),
+				'process.env.FORCE_REDUCED_MOTION': JSON.stringify(
+					!! process.env.FORCE_REDUCED_MOTION || false
+				),
+				__i18n_text_domain__: JSON.stringify( 'full-site-editing' ),
+				global: 'window',
+			} ),
```

Comment:
Had to copy over all the globals defined in `@automattic/calypso-build` so that we could add one more, the `__i18n_text_domain__` constant. I wonder if you're allowed to have multiple instances of `DefinePlugin` 🤔

Reply:
> I wonder if you're allowed to have multiple instances of `DefinePlugin` 🤔

You are :)